lib/CMakeLists.txt

```diff
@@ -33,7 +33,16 @@ endif()
 
 add_subdirectory(discord)
 
-include(libRocket.cmake)
+find_package(libRocket QUIET)
+if (${libRocket_FOUND})
```

Comment:
There should be an option to explicitly specify which behaviour to use to make it consistent for CI builds. Since we require a specialized version of libRocket it would make sense to have the default be "build from source" and have a toggle with which the usage of the system libraries could be forced.